Sources/XMTPiOS/Client.swift

```diff
@@ -100,23 +110,48 @@ public final class Client: Sendable {
 	}
 
 	static func create(account: SigningKey, apiClient: ApiClient, options: ClientOptions? = nil) async throws -> Client {
-		let privateKeyBundleV1 = try await loadOrCreateKeys(for: account, apiClient: apiClient, options: options)
+		let (privateKeyBundleV1, source) = try await loadOrCreateKeys(for: account, apiClient: apiClient, options: options)
+
+		let v3Client: FfiXmtpClient?
+
+		if options?.enableAlphaMLS == true && options?.api.env == .local {
+			let dbURL = URL.documentsDirectory.appendingPathComponent("xmtp-\(account.address).db3")
+			v3Client = try await LibXMTP.createClient(
+				logger: XMTPLogger(),
+				host: GRPCApiClient.envToUrl(env: apiClient.environment),
+				isSecure: apiClient.environment != .local,
+				db: dbURL.path,
+				encryptionKey: nil,
```

Comment:
I think it's important to encrypt the database. You can see how I did it here: https://github.com/xmtp/xmtp-android/blob/main/library/src/main/java/org/xmtp/android/library/Client.kt#L300-L322

Reply:
What do you think of allowing a user to pass it in instead of generating it for them?

Reply:
It will fail if the encryption key isn't the same every time they login or build with bundle. Feels like we should store it for them.

Reply:
I feel like persistence of secrets is more of an app level concern, I don't want to be prescriptive about how folks use the keychain since they might also want to load keys from elsewhere.